desktop/packages/linux/rs.ruffle.Ruffle.metainfo.xml

```diff
@@ -190,6 +190,7 @@
         <keyword xml:lang="ko">플레이어</keyword>
         <keyword xml:lang="nl">speler</keyword>
         <keyword xml:lang="pl">odtwarzacz</keyword>
+        <keyword xml:lang="pt-BR">jogador</keyword>
```

Comment:
I think this means "foot ball player" (participant) rather than "movie player" (presenter).